### ISSUE TITLE
MacOS: replace-then-find does not work in headlines

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#757]
On MacOS only, after finding an occurrence in a headline Leo switches to the log with "no text selected".

This bug first appeared in the Leo 5.7b1 release.

**Comment by karstenw** [comment]
My email report was unprecise:

the bug happens at replace-then-find.

The setup:
![screen 2018-03-03 um 10 46 06](https://user-images.githubusercontent.com/652583/36932932-4a21da26-1ed1-11e8-8bd8-2fa0ce007582.png)

Press cmd-g (find next):
![screen 2018-03-03 um 10 46 19](https://user-images.githubusercontent.com/652583/36932939-5ed7da10-1ed1-11e8-820d-aba536e34540.png)

Press cmd-alt-j (replace then find):
![screen 2018-03-03 um 10 48 29](https://user-images.githubusercontent.com/652583/36932949-8052f328-1ed1-11e8-8e94-a1a4dfdf5ad4.png)

**Comment by edreamleo** [comment]
I am unable to duplicate this behavior on the Mac, regardless of the value of the `@bool` minibuffer_find_mode setting.  Indeed, the find command selects text as expected.

The picture you give clearly shows that no headline text is selected, so it doesn't help much.

**Comment by karstenw** [comment]
I tried a short-as-possible screencast to illustrate the problem.
[leobug.mov.zip](https://github.com/leo-editor/leo-editor/files/1881170/leobug.mov.zip)

**Comment by edreamleo** [comment]
On Thu, Apr 5, 2018 at 1:06 PM, Karsten Wolf <notifications@github.com>
wrote:

> I tried a short-as-possible screencast to illustrate the problem.
> leobug.mov.zip
> <https://github.com/leo-editor/leo-editor/files/1881170/leobug.mov.zip>
>

​Thanks for this.  It explains the problem clearly.

This is a known issue, and can not easily be fixed.

The workaround is to use the indicated keystroke rather than actually
pushing the button. Indeed, it would likely be best to replace the buttons
with text, explaining which keystrokes to use.

Edward

**Comment by karstenw** [comment]
The buttons I used only for the screencast. On my machine, using keyboard shortcuts, yields the same result.

**Comment by edreamleo** [comment]
Git bisect reports this as the first bad commit:
```
commit 07c533e095eb2a9bfcc83bdccb31f66b883bccc3
Author: Edward K. Ream <edreamleo@gmail.com>
Date:   Tue Mar 28 10:43:23 2017 -0500

    Added support for --trace-focus and fixed a focus problem.  All tests pass.
    
    Previously Alt, Alt-Tab, Alt-Tab effectively lost focus.
    The first (bare) Alt puts focus in menu bar, and then
    double Alt-Tab then did not restore focus to menu properly.
    
    The fix was experimental code in qt_gui.onActivateEvent.
    
    Leo build: 20170328104323
```
This is the kind of code that could indeed cause this bug.

**Comment by edreamleo** [comment]
Fixed at ba34064, pretty much by brute force.

**Comment by edreamleo** [comment]
On Sat, Apr 7, 2018 at 4:22 AM, Karsten Wolf <notifications@github.com>
wrote:

> It works with the fix.
>
​Thanks for the confirmation.​


​Edward​
